src/Microsoft.ML.TensorFlow/TensorflowUtils.cs

```diff
@@ -181,7 +181,7 @@ internal static void DownloadIfNeeded(IHostEnvironment env, string url, string d
             using (var ch = env.Start("Ensuring meta files are present."))
             {
                 var ensureModel = ResourceManagerUtils.Instance.EnsureResourceAsync(env, ch, url, fileName, dir, timeout);
-                ensureModel.Wait();
+                ensureModel.Wait(5 * 60 * 1000); // 5 minute timeout period
```

Comment:
I think the TF models are around ~200MB. Is 5min too low for slower connections? For a 1Mb (mega-bit) connection, this would be 27min. 

How likely is a user to see the exception (`$@"Please copy the model file '{name}' from '{url}' to '{directory}'.");` sent to the channel? I think AutoML (API, CLI, & ModelBulder) will log them to file (not displayed), and would count them as a failed iteration and move on to the next iteration.